[Back to Wet Lab Protocols](MBARI_wet_lab.md)

# Environmental DNA (eDNA) extraction using Qiagen DNeasy blood and tissue kit V.1

#### Collin Closek<sup>1</sup>,Anni Djurhuus<sup>2</sup>,Katie Pitz<sup>3</sup>,Ryan Kelly<sup>4</sup>,Reiko Michisaki<sup>3</sup>,Kristine Walz<sup>3</sup>,Hilary Starks<sup>1</sup>,Francisco Chavez<sup>3</sup>,Alexandria Boehm<sup>1</sup>,Mya Breitbart<sup>2</sup>
#### <sup>1</sup>Center for Ocean Solutions, Stanford University, CA;<sup>2</sup>University of South Florida, College of Marine Science, St Petersburg, FL;<sup>3</sup>Monterey Bay Aquarium Research Institute, Moss Landing, CA;<sup>4</sup>University of Washington, Seattle, WA


## ABSTRACT
Nucleic acids extraction from the filters using the Qiagen DNeasy Blood and Tissue Kit with some modifications to the manufacturer’s protocol. These extraction protocols are used by the University of South Florida (USF) for Florida Keys samples (FK) and the Monterey Bay Aquarium Research Institute (MBARI) for Monterey Bay, CA samples (MB). FK samples were extracted in the Breitbart Laboratory at the University of South Florida and all MB samples were extracted in the Chavez Laboratory at the Monterey Bay Aquarium Research Institute using the same extraction protocol.

#### Preparation
1. Nucleic acids were extracted from the filters using the Qiagen DNeasy Blood and Tissue Kit with some modifications to the manufacturer’s protocol.
2. Prior to extraction 0.5 mm and 0.1 mm glass beads (BioSpec Products) were ashed at 500 °C for 5 hours.
3. 0.25 g of each size was distributed into 2.0 ml extraction tubes, the tubes were subsequently autoclaved and UV treated for 30 min.

#### Extraction
4. Sample filters were transferred to bead tubes with subsequent bead beating steps as in Djurhuus et al., in press, and then [incubated] at 56 °C overnight with 720 μl Buffer ATL and 80 μl Proteinase K (Qiagen). Remaining steps followed the manufacturer’s protocol for the DNeasy Blood and Tissue Kit.
5. To serve as a control in each set of extractions, an additional empty tube with beads was carried through the process as an extraction blank.
6. Samples were subsequently diluted 1:10, split into three aliquots, and distributed to three separate research laboratories for marker-specific processing: 16S rRNA gene at the University of South Florida (secondary 16S amplification done at RTSF Genomics Core, MSU), 18S rRNA and COI gene at the Monterey Bay Aquarium Research Institute (secondary COI amplification at RTSF Genomics Core), and 12S rRNA gene at Stanford University. See "Environmental DNA (eDNA) metabarcoding Illumina MiSeq NGS PCR Protocol" for 12S, 18S, 16S, and CO1.
7. Additionally, each loci had a mock community that served as positive controls for library preparation steps and sequencing.

[Back to Wet Lab Protocols](MBARI_wet_lab.md)
